pkg/golinters/usetesting/testdata/usetesting_configuration.go

```diff
@@ -0,0 +1,27 @@
+//golangcitest:args -Eusetesting
+//golangcitest:config_path testdata/usetesting_configuration.yml
+package testdata
+
+import (
+	"os"
+	"testing"
+)
+
+func Test_osMkdirTemp(t *testing.T) {
+	os.MkdirTemp("", "")
+}
```

Comment:
I'm unsure why we don't find it here

`func Test_osMkdirTemp`

Reply:
because it's disabled.